cinema_service/settings.py

```diff
@@ -86,12 +85,26 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.getenv("POSTGRES_DB", "default_db"),
+        "USER": os.getenv("POSTGRES_USER", "default_user"),
+        "PASSWORD": os.getenv("POSTGRES_PASSWORD", "default_password"),
+        "HOST": os.getenv("POSTGRES_HOST", "localhost"),
+        "PORT": os.getenv("POSTGRES_PORT", "5432"),
     }
 }
 
 
+if not all(os.getenv(var) for var in [
+    "POSTGRES_DB",
+    "POSTGRES_USER",
+    "POSTGRES_PASSWORD",
+    "POSTGRES_HOST",
+    "POSTGRES_PORT"
+]):
+    raise EnvironmentError("Required environment variables are not set.")
```

Comment:
Code Quality: It's good that you're checking if all necessary environment variables are set. However, the current implementation will raise an error if even a single variable is not set. It might be more user-friendly to specify which particular variable is missing in the error message.